### PR TITLE
Add a changelog file

### PR DESCRIPTION
**Pull request by Flaburgan**
Even if we don't do official release. I hate projects that don't have a changelog. So hard to follow what's really going on...

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## 0.5
```
Comment (Flaburgan) [inline Changelog.md]: I picked that arbitrary btw, feel free to change to any 0.x if you want to :)

**Comment by jaywink** [comment]
The trouble is, without releases, the changelog will just end up being a huge long list of changes etc that are pretty much as useful as browsing the commit log - and I'm not sure it's worth the extra overhead, to start adding links to PR's and such?

**Comment by Flaburgan** [comment]
Well introducing a changelog was indeed linked to the fact that we should start doing releases :p 
Not sure it's worth linking to PRs etc, just be able to now what's going on (I was confused by the introduction and then removing of sqlite for example). So I was thinking about small release when something important change (like big dependencies like switch from jade to nunchuck for example, or mysql to sqlite)

**Comment by jaywink** [comment]
I'm not really convinced this repo needs to do releases. After all, we're not shipping anything. We're not encouraging people to run the software, since that would not create any benefit. And I deploy at will on any merge to master ;)

How about auto-generating a changelog from the PR's? Still, the same info is easily available from the closed PR's. Without releases the changelog loses all of it's meaning.

**Comment by Flaburgan** [comment]
Fair enough, closing ;)
